docs/release_notes/v0.9-alpha.md

```diff
@@ -0,0 +1,53 @@
+# Spice v0.9-alpha (Mar 14, 2024)
+
+The v0.9 release adds several data connectors including the Spice data connector for the ability to connect to other `spiced` instances. Improved observability for `spiced` has been added with the new `/metrics` endpoint for monitoring deployed instances.
+
+## Highlights in v0.9-alpha
```

Comment:
Should we add Flight SQL server as a highlight?

Reply:
Done